src/DropdownButton/DropdownButton.less

```diff
@@ -58,35 +58,25 @@
  */
 
 .Button--primary.DropdownButton--toggle {
-  .DropdownButton--Caret--container {
-    .border--left--s(shade(@primary_blue, @shadeStepAmount * 2));
-  }
+  .border--left--s(shade(@primary_blue, @shadeStepAmount * 2));
```

Comment:
we can get rid of all these now since it's simpler to just have the top-level toggle button expose it's border instead of repeating the Button styles here on the inner.

i.e. remove the `border-left: 0` on line 24, then remove all of these except the disabled state below on L72, since that still needs to override the default button styles.

should probably also  get rid of the transparent border setting on L80 below as well, since that was only needed because of the awkward short partition showing up next to the darkened background of the primary action button